### DISCUSSION TITLE
[OFF-TOPIC QUESTIONS]

### DISCUSSION THREAD

**Discussion by busslina** [GregoryConrad/rearch-dart#37] [General]
Firstly, I know I do many questions, so feel free to not answer them or to take any time to answer.

Secondly, I wanted to know If I can make here some Off-topic questions (I would tag it with `[OFF-TOPIC]` in the title).

Thanks

**Comment by GregoryConrad** [comment]
Made a new discussion category:

https://github.com/GregoryConrad/rearch-dart/discussions/categories/off-topic
